dask/array/core.py

```diff
@@ -1220,6 +1228,11 @@ def modf(x):
 modf.__doc__ = np.modf
 
 
+@wraps(np.around)
+def around(x, decimals=0):
+    return map_blocks(x, partial(np.around, decimals=decimals), dtype=x.dtype)
```

Comment:
Or rather, now that I look below and see that you know about elemwise, why didn't elemwise work here?